src/components/ProfileInfos/index.js

```diff
@@ -158,11 +194,87 @@ const useStyles = makeStyles(theme => ({
     span: {
         fontFamily: `'Poppins', sans-serif;`,
         fontWeight: 'bold'
+    },
+    label:{
+        fontSize:20,
+        marginLeft:'60px',
+        marginBottom:20,
+        '@media(max-width: 850px)': {
+            fontSize:15,
+            marginLeft:'20px',
+            marginBottom:20,
+        },
+        '@media(max-width: 448px)': {
+            fontSize:15,
+            marginLeft:'10px',
+            marginBottom:20,
+        },
+        '@media(max-width: 375px)': {
+            fontSize:15,
+            marginLeft:'20px',
+            marginBottom:20,
+        },
+        '@media(max-width: 320px)': {
+            fontSize:10,
+            marginLeft:'10px',
+            marginBottom:20,
+        },
+        '@media(max-width: 360px)': {
+            fontSize:12,
+            marginLeft:'10px',
+            marginBottom:20,
+        },
+
+    },
+    checkboxField:{
+        marginLeft:20,
+
     }
 }))
 
+
+
 const ProfileInfos = () => {
     const classes = useStyles();
+
+    const[inputFname, setInputFname] = useState("")
+    const[inputLname, setInputLname] = useState("")
+    const[telephone, setTelephone] = useState("")
+    const[email, setemail] = useState("")
+    const[origin, setOrigin] = useState("")
+    const[gender, setGender] = useState("")
+    const[identification_type, setIdentification_type] = useState("")
+    const[identification_number, setIdentification_number] = useState("")
+    const[imageUrl, setImageUrl] = useState("")
+   
+
+    const getInfo = async ()=>{
+
+        const userId = localStorage.getItem("userProfile");
+        try {
+            const response = await fetch('http://localhost:5000/user/'+ userId );
+
```

Comment:
Don't hardcode the url in code because later in production this will not work, please consider making this `http://localhost:5000` a variable in `.env` then use it here instead

Reply:
The problem now is fixed